frontend/components/AdminItemsTable.js

```diff
@@ -49,7 +49,7 @@ export default function AdminItemsTable({ registry }) {
 
   // deletes item and updates our cache
   const handleDeletion = id => {
-    mutate("/registry/admin", async items => {
+    mutate(["/registry/admin", accessToken], async items => {
```

Comment:
Yup, that's the correct key.

However, the `/registry/admin` endpoint returns a registry object, not an items array though
```jsx
mutate(["/registry/admin", accessToken], async items => {
mutate(["/registry/admin", accessToken], async registry => {
```
You can still access the items through `registry.items` though.

Reply:
And because we can access the cached registry like that, we don't even need to pass the registry object as a prop to `AdminItemsTable`.

Because the only place we use the registry is inside that mutate.

Instead, it would be simpler to just pass down items as a prop.